app/Providers/Blocks/OurReportingGridServiceProvider.php

```diff
@@ -31,7 +31,9 @@ public function render($block, $content = '', $is_preview = false, $post_id = 0)
         $context = Timber::get_context();
 
         $context['block']['heading'] = get_field('heading');
+        $context['block']['content_type'] = get_field('content_type');
```

Comment:
Without a default value set no content is displayed:

![CleanShot 2024-04-05 at 16 58 38](https://github.com/open-contracting-partnership/website/assets/84081/9e442ed1-91b8-4395-92d7-2cb7cf734cba)

You'll need something like `get_field('content_type') ?? 'card-strapline'` to ensure this value is used as a fallback.

Reply:
Ahh got it, sorry I missed that during the switch from a true/false to radio buttons